Kconfig

```diff
@@ -0,0 +1,23 @@
+menu "nkolban's c++ wrappers"
+
+menuconfig NKOLBAN
+    bool "nkolban's c++ wrappers"
+    default y
+    help
+        Select this option to use nkolban's c++ wrappers.
+
+menuconfig NKOLBAN_BLE
+    bool "bluetooth wrappers"
+    depends on BT_ENABLED && NKOLBAN
+    default y
+    help
+        Select this option to use nkolban's c++ bluetooth ble wrappers.
+
+menuconfig NKOLBAN_BLE2902
+    bool "BLE2902"
+    depends on NKOLBAN_BLE
+    default y
+    help
+        Select this option to use nkolban's c++ bluetooth BLE2902 wrappers.
+
+endmenu
```

Comment:
Can you clarify what the intent is of these options and the impact of not having them?

Reply:
See component.mk

It's less than half done, but it's a start of being able to use your code in my project - I'm only using the ble part for now.

My use-case is to git-clone your repo to some local directory and create a symlink to it inside the components directory of each of my esp-idf projects that use it (1 for now).

Is there a better way to do this, outside of merging it into esp-idf?

I was going to email you about this but then that commit came along and caused compilation errors.
I thought I'd save you the time, plus they're backwards compatible :)